postprocessing/pp-h264.c

```diff
@@ -78,7 +76,11 @@ int janus_pp_h264_create(char *destination) {
 	vStream->time_base = (AVRational){1, 90000};
 	vStream->codec->width = max_width;
 	vStream->codec->height = max_height;
+#if LIBAVCODEC_VER_AT_LEAST(53, 0)
+	vStream->codec->pix_fmt = AV_PIX_FMT_YUV420P;
+#else
```

Comment:
There are many generic fixes there. This one of the exmaple. But without it I coud not compile it on Windows